vignettes/intro.Rmd

```diff
@@ -44,58 +46,79 @@ In the `dummy_data`, we have the following columns:
 1. `y` is the continuous outcome.
 1. `y_b` is the binary outcome.
 
-## Obtain Treatment Effect for Continuous Outcome
+## Obtain Treatment Effect for Continuous Outcomes using the general variance
 
-In simplest case of a continuous outcome, and we want to adjust for `covar`, `s1`, where `covar` is
-a covariate and `s1` is a stratification factor.
-Randomization is permuted-block randomization stratified by `s1`.
-A proper model to use is `y ~ treatment * s1 + covar`.
+For the continuous outcome `y`, the linear model includes `covar` as a covariate,
+`s1` as a stratification factor. The randomization scheme is a permuted-block
+randomization stratified by `s1`. The model formula also includes the treatment
+by stratification interaction as `y ~ treatment * s1 + covar`.
 
 ```{r}
-robin_glm(y ~ treatment * s1 + covar, data = dummy_data, treatment = treatment ~ pb(s1))
+robin_lm(y ~ treatment * s1 + covar,
+  data = dummy_data,
+  treatment = treatment ~ pb(s1)
+)
 ```
 
 We can also use the Huber-White variance estimator by setting `vcov = "vcovHC"`.
-Please note that in this case, the model formula should not contain treatment interaction with stratification factors.
+Please note that in this case, the model formula should not contain the
+treatment by stratification (covariate) interaction.
 
 ```{r}
-robin_glm(y ~ treatment + s1 + covar, data = dummy_data, treatment = treatment ~ pb(s1), vcov = "vcovHC")
+robin_lm(y ~ treatment + s1 + covar,
```

Comment:
the default link function in robin_glm is gaussian so it is worth mentioning that robin_glm also works

Reply:
Agree and added.